### PR TITLE
Adjust existing function to accept kubescheduler as well

### PR DESCRIPTION
**Pull request by kasturinarra**
Adjust existing function so that it accepts both kubescheduler & scheduler CRD.

### CODE REVIEW

File: features/step_definitions/scheduler.rb [hunk 1]
```diff
@@ -7,11 +7,11 @@
   }
 end
 
-Given /^the #{QUOTED} scheduler CR is restored after scenario$/ do |name|
+Given /^the #{QUOTED} #{QUOTED} scheduler CR is restored after scenario$/ do |name, crd|
```
Comment (liangxia) [inline features/step_definitions/scheduler.rb:10]: How about `Given /^the CR #{QUOTED} named #{QUOTED} is restored after scenario$/ do |type, name|`

Comment (kasturinarra) [reply features/step_definitions/scheduler.rb:10]: should be fine i guess, i will update it accordingly

Comment (kasturinarra) [reply features/step_definitions/scheduler.rb]: Addressed

**Comment by kasturinarra** [comment]
@zhouying7780 @liangxia could you please help review ? Thanks!!

**Pull request merged**